src/card_operations/card_basic.c

```diff
@@ -0,0 +1,370 @@
+/**
+ * @file    card_basic.c
```

Comment:
filename `card_basic` can be replaced with `card_internals`

Reply:
The comment should say rename